vike/client/client-routing-runtime/prefetch.ts

```diff
@@ -138,3 +215,24 @@ async function prefetchIfPossible(url: string): Promise<void> {
   if (!(await isClientSideRoutable(pageContextFromRoute._pageId, pageContext))) return
   await prefetchAssets(pageContextFromRoute._pageId, pageContext)
 }
+
+async function prefetchContextIfPossible(url: string, expire: number | undefined): Promise<void> {
+  const now = Date.now()
+  const lastPrefetch = globalObject?.lastPrefetchTime?.get(url)
+  if (lastPrefetch && expire && now - lastPrefetch < expire) {
+    return
+  }
+  const pageContext = await createPageContext(url)
+  let pageContextFromRoute: PageContextFromRoute
+  try {
+    pageContextFromRoute = await route(pageContext)
```

Comment:
`route()` is being called twice: by `prefetchAssetsIfPossible()` and `prefetchContextIfPossible()`. Can we run it only once instead? It would be nice to avoid calling the user's Route Functions too much.